docs/user-guides/anonymous-access.md

```diff
@@ -6,7 +6,7 @@ Bypass identity verification or fall back to anonymous access when credentials f
   <summary>
     <strong>Authorino capabilities featured in this guide:</strong>
     <ul>
-      <li>Identity verification & authentication → <a href="./../features.md#anonymous-access-authenticationanonymous">Anonymous access</a></li>
+      <li>Identity verification & authentication → [Anonymous access](./../features.md#anonymous-access-authenticationanonymous)</li>
```

Comment:
I believe the reason why this is broken in the Docs website is not the HTML vs MD format, but because of the relative path – which BTW currently does not break the link when the page is rendered in Github:

<img width="1536" alt="Screenshot 2024-07-06 at 03 35 04" src="https://github.com/Kuadrant/authorino/assets/1842261/ad4b0a86-3e5d-411e-976d-f0b7ce18a2b4">
<img width="1536" alt="Screenshot 2024-07-06 at 03 32 45" src="https://github.com/Kuadrant/authorino/assets/1842261/e9125f28-e875-4a57-867f-30b7e081951c">

<br/><br/>

Using MD format does NOT work in Github in this case where the content is still wrapped within a HTML tag (`details > summary > ul > li`).

Before this PR:
<img width="1100" alt="Screenshot 2024-07-06 at 03 29 29" src="https://github.com/Kuadrant/authorino/assets/1842261/e4ce1b33-c2c1-493f-b6a1-494482191d80">


After this PR:
<img width="1100" alt="Screenshot 2024-07-06 at 03 29 23" src="https://github.com/Kuadrant/authorino/assets/1842261/ffe06ed9-3074-4b97-bd00-ff4575eb6110">

Ideally, we'd find a fix that makes it work both in Github and in the Docs website. Maybe try something focused on the relative path?

Reply:
If it helps, we can change to avoid this pattern of an anchor link within a detail summary. It's probably cumbersome for the user anyway; depending if click on the link or its surrounding you either follow the link or open the details box.

I think it's repeated at all Authorino User guides, so some work to do, but probably worthy. WDYT?

Reply:
That would probably help. Tedious to do, but might be worth it. Happy to help with it

Reply:
OK done - works good for both mkdocs rendered content + GH.